Doc/library/asyncio-stream.rst

```diff
@@ -367,13 +366,177 @@ Stream
 
```

Comment:
Please update *high level API index* with the new APIs.

https://docs.python.org/3.8/library/asyncio-api-index.html

Reply:
Thanks, I have added the methods and classes added to the index. Can we also recommend in the deprecated methods about new replacements? Like `open_connection` could have a sentence that `connect` is recommended. I have used pretty much same sentences since they do the same and recommending would avoid confusion over duplicate functions.

Reply:
Yes! Please do that.